components/automate-ui/src/app/entities/policies/policy.requests.ts

```diff
@@ -28,10 +22,6 @@ export class PolicyRequests {
 
   constructor(private http: HttpClient) { }
 
-  public getIamVersion(): Observable<IamVersionResponse> {
-    return this.http.get<IamVersionResponse>(`${env.auth_v2_url}/policy_version`);
```

Comment:
Should similarly remove `policy_version` calls from components/automate-ui/e2e/admin.e2e-spec.ts